pkg/crc/services/dns/template.go

```diff
@@ -8,9 +8,7 @@ import (
 )
 
 const (
-	dnsmasqConfTemplate = `user=root
-port= {{ .Port }}
-bind-interfaces
+	dnsmasqConfTemplate = `listen-address={{ .IP }}
```

Comment:
Could it be localhost instead of the external IP?

Reply:
Dunno if listen-address is mandatory?

Reply:
> Dunno if listen-address is mandatory?

If we not specify then it will also listen to loopback interface.

```-a, --listen-address=<ipaddr>
              Listen on the given IP address(es). Both --interface and --listen-address options may be given, in which case the set of both interfaces and addresses is used. Note that if no --interface option is  given,  but  --listen-address  is,
              dnsmasq will not automatically listen on the loopback interface. To achieve this, its IP address, 127.0.0.1, must be explicitly given as a --listen-address option.
```

Reply:
> there are most likely some 10.x address we can use that would be reachable by all pods but not exposed externally.

May be we can attach to a dummy interface but didn't try it since it is only available for openshift bundle right now and not with microshift one.